packages/asset-service/src/service/AssetService.ts

```diff
@@ -76,6 +76,7 @@ export class AssetService {
    */
   async initialize() {
     try {
+      if (!this.assetFileUrl) throw new Error('No assetFileUrl')
```

Comment:
Is it worth try to parse it as a valid URL and throw an error if not? Probably not.

Reply:
Yeah I was thinking about that as well. I'm not opposed to doing so but might be overkill in this case.